whipper/common/mbngs.py

```diff
@@ -274,6 +274,19 @@ def _getMetadata(release, discid, country=None):
 # see http://bugs.musicbrainz.org/browser/python-musicbrainz2/trunk/examples/
 #     ripper.py
 
+def getReleaseMetadata(release_id):
+    import musicbrainzngs
+
+    musicbrainzngs.set_useragent("whipper", whipper.__version__,
+                                 "https://github.com/whipper-team/whipper")
```

Comment:
I will work on this and think of a better solution. Thanks a lot !

Reply:
I think the `set_useragent` can be solved by putting `import musicbrainzngs` then `set_useragent` at the beginning of this file. 

But that the latter part of this function is still duplicative.

Reply:
And also should we import only `whipper.__version__` instead of  `whipper`? 
Since from what i see only `whipper.__version__` is used in the file.
```
from whipper import __version__ as VERSION
```

Reply:
Should i move all [this block](https://github.com/whipper-team/whipper/blob/6e3e21f9d5466667a2ac21c7f470b90f4c0ad35b/whipper/common/mbngs.py#L319L336) of code into the new `getReleaseMetadata` function ? Since the rest is still usable (mostly for logging and writing record). With this and the one i suggested above, the code should no longer be duplicative.

Reply:
> And also should we import only `whipper.__version__` instead of `whipper`?

Yes (maybe in another commit).